.circleci/config.yml

```diff
@@ -112,7 +114,10 @@ jobs:
           destination: coverage.txt
 
   run-windows-unit-test:
-    executor: win/default
+    executor:
```

Comment:
I don´t know if it's possible, but I see that the same executor config is defined across all the different jobs. Could we add the executor definition into top `executors` (line 39) and just [reference it on the rest](https://circleci.com/docs/configuration-reference/#executors). This way when we change it (version or whatever) we just need to change it on one part